pkg/kotsadmsnapshot/restore.go

```diff
@@ -127,6 +127,10 @@ func CreateApplicationRestore(ctx context.Context, kotsadmNamespace string, snap
 		return errors.Wrap(err, "failed to create restore")
 	}
 
+	// TODO(improveddr)
```

Comment:
i don't think this todo is correct. restore objects for EC don't get created in KOTS. they're created by the EC CLI. so nothing in here needs to change

Reply:
removed